stor/dx.py

```diff
@@ -82,9 +75,15 @@ def _dx_error_to_descriptive_exception(client_exception):
 
 
 @contextmanager
-def _propagate_dx_exceptions():
+def _wrap_dx_calls():
     """Bubbles all dxpy exceptions as `DNAnexusError` classes
     """
+    auth_token = settings.get()['dx']['auth_token']
+    if auth_token:  # pragma: no cover
```

Comment:
presumably this code ought to be covered.  pretty easy to use this with no-op credentials